owrx/socket.py

```diff
@@ -2,7 +2,7 @@
 
 
 def getAvailablePort():
-    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s = socket.socket(socket.AF_INET6, socket.SOCK_STREAM)
```

Comment:
This method has nothing to do with the webserver, it is used for internal communication. I'm not sure if all affected tools support IPv6 yet.

Reply:
I will check if it works without this change. If yes, I will revert this line.